packages/controllers/src/snaps/SnapController.ts

```diff
@@ -580,26 +580,25 @@ export class SnapController extends BaseController<
         this._maxIdleTime &&
         timeSince(runtime.lastRequest) > this._maxIdleTime
       ) {
-        this._stopSnap(snapId);
+        this._stopSnap(snapId, SnapStatusEvent.stop);
       }
     });
   }
 
-  _onUnresponsiveSnap(snapId: SnapId) {
-    this._transitionSnapState(snapId, SnapStatusEvent.crash);
-    this._stopSnap(snapId, false);
+  async _onUnresponsiveSnap(snapId: SnapId) {
+    await this._stopSnap(snapId, SnapStatusEvent.crash);
     this.addSnapError({
-      code: -32001, // just made this code up
```

Comment:
I've confirmed that nothing relied on this code. We shouldn't add non-standard error codes until we have established a standard of our own.